src/main/java/com/rabbitmq/client/impl/recovery/DefaultRetryHandler.java

```diff
@@ -40,19 +40,19 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultRetryHandler.class);
 
-    private final BiPredicate<? super RecordedQueue, Exception> queueRecoveryRetryCondition;
-    private final BiPredicate<? super RecordedExchange, Exception> exchangeRecoveryRetryCondition;
-    private final BiPredicate<? super RecordedBinding, Exception> bindingRecoveryRetryCondition;
-    private final BiPredicate<? super RecordedConsumer, Exception> consumerRecoveryRetryCondition;
+    protected final BiPredicate<? super RecordedQueue, Exception> queueRecoveryRetryCondition;
```

Comment:
allow consumers to more easily extend this class and be able to reuse the existing logic